features/joinroom/impl/src/main/kotlin/io/element/android/features/joinroom/impl/JoinRoomPresenter.kt

```diff
@@ -56,15 +75,29 @@ class JoinRoomPresenter @AssistedInject constructor(
                     roomDescription.get().toContentState()
                 }
                 else -> {
-                    ContentState.Loading(roomId)
+                    coroutineScope.launch {
```

Comment:
No need to use a coroutineScope, `produceState` is already suspending

Reply:
Done